tasks/agent.py

```diff
@@ -109,6 +109,10 @@ def build(ctx, rebuild=False, race=False, build_include=None, build_exclude=None
 
     if sys.platform == 'win32':
         windres_target = "pe-x86-64"
+
+        # Important for x-compiling
+        env["CGO_ENABLED"] = "1"
```

Comment:
if this is to make the x86 compile work, I would move it inside the `if arch == "x86":` below, just to make sure there aren't any unintended consequences on other platforms.